content/rc/security/access-control/data-access-control/create-assign-users.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: Create and edit database users
+LinkTitle: Create users
+description: Create a database user and assign it a role.
+weight: 25
+alwaysopen: false
+toc: "true"
+headerRange: "[1-3]"
+categories: []
+aliases: 
+---
+
+Before you create a database user, you must [create a data access role]({{<relref "rc/security/access-control/data-access-control/create-roles">}}) to assign to that user.
+
+## Create a user
+
+To create a user:
+
+1. Go to **Data Access Control > Users**.
+
+1. Select the `+` button to create a new user.
+
+1. Enter the username.
+
+1. Select a [**Role**]({{<relref "rc/security/access-control/data-access-control/create-roles">}}) from the list.
+
+1. Enter and confirm the user's password.
+
+1. Select the check mark to save the user.
+
+## Assign roles to existing users
+
+To assign a data access role to an existing user:
+
+1. Go to **Data Access Control > Users**.
+
+1. Point to the user and select the pencil icon when it appears:
```

Comment:
Missing pencil icon. I would either change the colon to a period or add the pencil icon.
```suggestion
1. Point to the user and select the pencil icon when it appears.
```